alf/experience_replayers/replay_buffer.py

```diff
@@ -18,10 +18,78 @@
 import torch.nn as nn
 
 import alf
+from alf import data_structures as ds
 from alf.nest.utils import convert_device
 from alf.utils.data_buffer import atomic, RingBuffer
 
 
+def get_nest_attr(obj, path):
```

Comment:
Seems we already have this: `alf.nest.get_field()`

Reply:
Good points.  Done.  I originally tried it and somehow an error appeared..  Maybe some other bug caused it.